packages/interactive_media_ads/lib/src/platform_interface/platform_companion_ad.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// An object that holds data corresponding to the companion Ad.
+base class PlatformCompanionAd {
+  /// Constructs a [PlatformCompanionAd].
+  PlatformCompanionAd({
+    required this.apiFramework,
+    required this.height,
+    required this.resourceValue,
+    required this.width,
+  });
+
+  /// The API needed to execute this ad, or null if unavailable.
+  final String? apiFramework;
+
+  /// The height of the companion in pixels.
+  final int? height;
```

Comment:
What do null height/width mean?

Reply:
It means that it is unavailable: https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side/api/reference/com/google/ads/interactivemedia/v3/api/CompanionAd#getWidth(). I change the zero to null to avoid foot guns. I'll update docs.